meteor/client/lib/editAttribute/edit-attribute-enum-dropdown.tsx

```diff
@@ -2,26 +2,19 @@ import { DropdownOption, EditAttributeDropdown } from './edit-attribute-dropdown
 import { IEditAttributeBaseProps } from './edit-attribute-base'
 import * as React from 'react'
 import { useMemo } from 'react'
+import { EnumLike } from '@sofie-automation/corelib/dist/deviceConfig'
```

Comment:
This is the way it's done with most of the files that export some types that both the frontend and the backend use. Since the backend, the frontend and the shared library live in a monorepo, I don't see a problem in importing them this way. Please suggest a better, concrete solution.

Reply:
I'm aware of how widespread it is in the codebase and unfortunately I don't have a good solution for it currently (except for completely separating the frontend and backend code). 
I would just like us to be aware of it, since this is not ideal.